robot/Crisis-Management/resources/locators_48.py

```diff
@@ -71,7 +71,14 @@
         "home_tab": "//a[@href='/TractionThrive/s/']",
         "staff_info": "//div/lightning-formatted-text[text()='{}']",
         "staff_contact_info": "//lightning-input[@data-field='{}']//input",
+    },
 
+    "community_template":{
+        "add_metrics": "//a[text()='{}']",
+        "template_button": "//button[@title='{}']",
+        "template_header": "//div[contains(@class,'cb-StartWizard-TemplateTitle')]/header[contains(text(),'{}')]",
+        "template_name"  : "//input[@id='cb-StartWizard-propertyInput-Name']",
+        "title" : "//*[text()='{}']",
```

Comment:
Suggestion: Can this same locator be used in place of add_metrics?

Reply:
@sgovindankutty gonna go ahead and merge but just tagging so you're aware in case you'd like to make this change at some point.